R Data Curation Primer/R-data-curation-primer.md

```diff
@@ -173,6 +173,14 @@ library(urltools)
 library(rgeolocate)
 ```
 
+##### Dependency management systems for R
```

Comment:
I think this looks good.  Thanks for incorporating this list of tools that help make code reproducible!

Reply:
Thanks both!